lib/index.js

```diff
@@ -83,40 +164,56 @@ function waitForAlive(port, delay) {
     if (Date.now() > start) {
       const result = nodeNC(port, 'ping\r\n');
       response = result.stdout && result.stdout.toString();
-      err = result.stderr && result.stderr.toString();
     }
   }
   if (response !== 'pong') {
     throw new Error(
       'Timed out waiting for sync-rpc server to start (it should respond with "pong" when sent "ping"):\n\n' +
-        err +
         '\n' +
         response
     );
   }
 }
 
 function nativeNC(port, input) {
-  return spawnSync('nc', [host, port], {
-    input: input,
-    windowsHide: true,
-    maxBuffer: Infinity,
-  });
+  index[0] = index[0] + 1;
+  const channel = 'nativeNC';
+  debugMode && logToFile(`${channel} received msg #${index[0]} at client: ${input}`, localClientLog);
+  let res;
+  try {
+    res = spawnSync(`nc`, [host, port], {
+        input: input,
+        maxBuffer: Infinity,
+        windowsHide: true,
+      });
+  } catch (e) {
+    debugMode && logToFile(`Received error: ${e.message}`);
```

Comment:
should we throw this error back to the caller or does this help with the crashing